### DISCUSSION TITLE
Welcome to fosdem-2022-devroom Discussions!

### DISCUSSION THREAD

**Discussion by pombredanne** [software-composition-analysis/fosdem-2022-devroom#2] [Announcements]
## 👋 Welcome!
We’re using this place to discuss the creation, organization and planning for FOSDEM 2022 **Software composition and dependency management** among organizers. 

To get started, feel free to post a quick comment below with a short introduction and what you want to be involved with.

@amottier @cnuel @maxhbr @maxhbr @mcjaeger @pombredanne : I am creating this so we can also discuss here as needed and put the details for our planning calls too.

**Comment by maxhbr** [comment]
Hey I am Maximilian Huber, and I am mainly unsure whether I have to introduce myself twice, since I am also mentioned twice.

**Comment by juliancoccia** [comment]
Hi everyone. Julian Coccia here presenting the SCANOSS project.
